packages/gatsby-theme-docs/src/hooks/use-code-example-language.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const CodeExampleLanguageContext = React.createContext({
+  language: '',
+  setLanguage: () => {},
+});
+CodeExampleLanguageContext.displayName = 'CodeExampleLanguageContext';
```

Comment:
This is wrong, the context is not a component

Reply:
You mean you prefer only components to have names?

If it is wrong, why is `displayName` a field you can set?

Reply:
This is JS, you can attach any field you want. The `displayName` is a static property used by the React devtools, for components.

Reply:
> The `displayName` is a static property used by the React devtools, for components.

Sounds like a React Context is in fact a component.

Reply:
`Context.Provider` and `Context.Consumer` are components, not `Context`.

<img width="296" alt="image" src="https://user-images.githubusercontent.com/1110551/80493516-d0712680-8965-11ea-83c3-15305eafd5a8.png">

<img width="535" alt="image" src="https://user-images.githubusercontent.com/1110551/80493572-dff06f80-8965-11ea-8a15-40f7dda9187e.png">

https://reactjs.org/docs/context.html#api